### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by furzeface** [code-computerlove/dotfiles#6]
Test on a new machine from scratch.

**Comment by furzeface** [comment]
@tawashley what did you think when you tried them on your new Mac yesterday? Any positive or negative feedback, additions we could make? Do you think it saved you some time?

**Comment by tawashley** [comment]
@furzeface Definitely helped get a clean mac setup pretty quickly. Not really a negative, but the `.osx.sh` being empty meant I had to spend abit of time going through to see what I actually wanted - but I did this whilst downloading applications. This is more something for https://github.com/code-computerlove/dotfiles/issues/3 though.

It may also benefit from adding something about expecting a OS X dialog box for installing XCode tools, as I wasn't sure for a second if I need to click 'install' to get things moving. 

Other than that, other additions were just preference (getting spectacle, alfred). Everything seemed good!
